packages/amplify_datastore/ios/Classes/types/query/QueryPaginationBuilder.swift

```diff
@@ -18,7 +18,7 @@ import Amplify
 
 public class QueryPaginationBuilder {
    
-    static func fromSerializedMap(_ serializedMap: [String: Any]?) -> QueryPaginationInput {
+    static func fromSerializedMap(_ serializedMap: [String: Any]?) -> QueryPaginationInput? {
         var page: UInt = 0, limit: UInt = QueryPaginationInput.defaultLimit;
```

Comment:
nit: can probably move `var page: UInt = 0, limit: UInt = QueryPaginationInput.defaultLimit;` into the `if let data = serializedMap` block.